include/dungeon_entity.h

```diff
@@ -101,7 +101,7 @@ struct DungeonEntityData
     /* 0x8C */ u8 IQSkillsSelected[4]; // IQ skills selected in the IQ skills menu.
     /* 0x90 */ u8 IQSkillsEnabled[4];
     /* 0x94 */ u8 tactic;
-    u8 fill95[0xA4 - 0x95];
+    u8 fill9C[0xA4 - 0x95];
```

Comment:
Is this rename accurate? This variable's offset hasn't changed.

Reply:
Forgot to change it back from a function I was trying to match (but didn't). Will change it back..

Reply:
Fixed.